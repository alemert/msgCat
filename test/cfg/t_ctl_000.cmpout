2013-01-15 14:09:56  27071 00100 SYS starting test/bin/gcc/64/Linux.x86_64/t_ctl_000
