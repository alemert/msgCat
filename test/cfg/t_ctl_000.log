2013-02-27 19:49:07  20080 00100 SYS starting t_ctl_000
