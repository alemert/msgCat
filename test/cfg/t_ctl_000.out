2013-01-15 14:49:27  30178 00100 SYS starting t_ctl_000
